_run/single/README.md

```diff
@@ -52,6 +52,14 @@ this example.
 |`OSEQ`|1|order sequence|
 |`PRICE`|10akash|price to bid|
 
+To get DNS routing to work locally, there are two addresses which will probably need to set to configure requests to hit the kind docker container. To route requests back to the local interface, add the following two lines to your `/etc/hosts` for the Akashd and Akash-Provider examples to work correctly.
+
+* `127.0.0.1   akashd.localhost`
+* `127.0.0.1   akash-provider.localhost`
+
+Or if it does not conflict with other local rules, use a wildcard for localhost:
+* `127.0.0.1   *.localhost`
```

Comment:
don't need these if using `systemd-resolved` or similar.  it's nice to have modern defaults.

```
$ cat /etc/hosts
# Static table lookup for hostnames.
# See hosts(5) for details.

127.0.0.1 localhost
::1       localhost
127.0.1.1 laozi.localdomain laozi

192.168.1.125 monolith
$ dig +short foobar.localhost
127.0.0.1

Reply:
That is nice that `systemd-resolved` handles it. This is one of those issues that dev's machine will handle a little differently though, I don't know what Mac's have available these days, so figured best document it with a fairly standard routing convention.

Reply:
it works fine on os x too.